src/components/elements/Container.js

```diff
@@ -1,14 +1,21 @@
 import styled from 'styled-components'
+import { theme } from 'theme'
 
 import Flex from './Flex'
 
-const Container = styled(Flex)``
+const Container = styled(Flex).withConfig({
+  shouldForwardProp: prop => !['px'].includes(prop)
+})`
+  ${theme({
+    flexDirection: 'column',
+    ml: 'auto',
+    mr: 'auto',
+    pt: [5, 5, 6, 6],
+    px: [3, 3, 0, 0]
+  })}
+`
 
 Container.defaultProps = {
-  flexDirection: 'column',
-  ml: 'auto',
-  mr: 'auto',
-  pt: [5, 5, 6, 6],
```

Comment:
**reminder:** This was used in quite a few places (e.g. `Container.defaultProps.pt`), need to replace with relevant value.